### PR TITLE
adds related articles, new videos & digital expo

### PR DESCRIPTION
**Pull request by will-wbd**

### CODE REVIEW

File: _includes/home/digital-expo.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                  <div class="usa-card__media usa-card__media--inset">
+                    <div class="usa-card__img">
+                      <div class="vha-video">
+                        <a href="https://www.youtube.com/watch?v=WZ1aCCGeMBs" class="vha-video__play vha-video__play-action stretched-link">
+                            <svg viewBox="0 0 46 46" fill="none" class="vha-video__play-icon">
+                                <path d="M23 0C10.2975 0 0 10.2971 0 23C0 35.7029 10.2975 46 23 46C35.7025 46 46 35.7029 46 23C46 10.2971 35.7025 0 23 0ZM19.534 14.0944C20.2076 13.7011 21.1 13.6666 21.9096 14.2094L30.3324 19.8927C32.7858 21.6913 32.859 24.242 30.3324 26.0935L21.9096 31.7768C20.2904 32.8624 18.4001 31.7515 18.4001 29.7091V16.2748C18.4001 15.2559 18.8601 14.4877 19.534 14.0944Z" fill="white"/>
+                            </svg>
+                        </a>
+                        <div class="vha-video__poster">
+                          <img src="assets/img/home/poster-our-mission-and-opportunity.jpg" class="radius-lg"  alt=""/>
```
Comment (ronald-wbd) [inline _includes/home/digital-expo.html:22]: alt text is missing here

File: _includes/home/digital-expo.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                  <div class="usa-card__media usa-card__media--inset">
+                    <div class="usa-card__img">
+                      <div class="vha-video">
+                        <a href="https://www.youtube.com/watch?v=FJ_bs6E9awY" class="vha-video__play vha-video__play-action stretched-link">
+                            <svg viewBox="0 0 46 46" fill="none" class="vha-video__play-icon">
+                                <path d="M23 0C10.2975 0 0 10.2971 0 23C0 35.7029 10.2975 46 23 46C35.7025 46 46 35.7029 46 23C46 10.2971 35.7025 0 23 0ZM19.534 14.0944C20.2076 13.7011 21.1 13.6666 21.9096 14.2094L30.3324 19.8927C32.7858 21.6913 32.859 24.242 30.3324 26.0935L21.9096 31.7768C20.2904 32.8624 18.4001 31.7515 18.4001 29.7091V16.2748C18.4001 15.2559 18.8601 14.4877 19.534 14.0944Z" fill="white"/>
+                            </svg>
+                        </a>
+                        <div class="vha-video__poster">
+                          <img src="assets/img/home/poster-our-strategic-advantage.jpg" class="radius-lg"  alt=""/>
```
Comment (ronald-wbd) [inline _includes/home/digital-expo.html:70]: alt text is missing here too

File: _includes/home/expertise-demand.html [hunk 3]
```diff
@@ -12,16 +12,30 @@ <h1 class="margin-top-0">Expertise on Demand</h1>
...
+                    </div>              
+                    <div class="tablet:grid-col-6 tablet-lg:grid-col-8 tablet-lg:grid-offset-4 clinical-training__video">
+                        <div class="vha-video">
+                            <a href="https://www.youtube.com/watch?v=siluLqx1TFI" class="vha-video__play vha-video__play-action stretched-link">
+                                <svg viewBox="0 0 46 46" fill="none" class="vha-video__play-icon">
+                                    <path d="M23 0C10.2975 0 0 10.2971 0 23C0 35.7029 10.2975 46 23 46C35.7025 46 46 35.7029 46 23C46 10.2971 35.7025 0 23 0ZM19.534 14.0944C20.2076 13.7011 21.1 13.6666 21.9096 14.2094L30.3324 19.8927C32.7858 21.6913 32.859 24.242 30.3324 26.0935L21.9096 31.7768C20.2904 32.8624 18.4001 31.7515 18.4001 29.7091V16.2748C18.4001 15.2559 18.8601 14.4877 19.534 14.0944Z" fill="white"/>
+                                </svg>
+                            </a>
+                            <div class="vha-video__poster">
+                                <img src="assets/img/home/poster-clinical-training-2.jpg" class="radius-lg" />
```
Comment (ronald-wbd) [inline _includes/home/expertise-demand.html:34]: alt text also missing

**Review by ronald-wbd — COMMENTED** [review]
Layout and functionality looks great! Just some missing alt texts...

**Comment by ecstech-admin** [comment]
Echo what Ronald said above.

My only suggestion (and I could go either way on this) would be to suggest making the headshots clickable for bios in addition to the "Learn More" button only because that's what many people are used to. Otherwise, if you're on mobile, the button is at the bottom after all the photos. A user may not get there and know that there's "more". I just imagine someone viewing the page and trying to click on photos and getting frustrated.

**Review by ecstech-admin — COMMENTED** [review]
Echo what Ronald said above.

My only suggestion (and I could go either way on this) would be to suggest making the headshots clickable for bios in addition to the "Learn More" button only because that's what many people are used to. Otherwise, if you're on mobile, the button is at the bottom after all the photos. A user may not get there and know that there's "more". I just imagine someone viewing the page and trying to click on photos and getting frustrated.

**Review by ronald-wbd — APPROVED** [review]
code and layout looks good!

**Pull request merged**